### DISCUSSION TITLE
blog/Etc/git-flow

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#651] [Announcements]
# blog/Etc/git-flow

git-flow는 브랜치를 5가지 종류로 구분하여 개발하는 전략이다. git-flow의 브랜칭 전략 주기적으로 배포를 진행해야 하는 프로젝트에 적합; 다양한 분야의 사람들과 개발할 때 효과적; 다양한 상황에 유연하게 대응; git-flow의 흐름 main 배포 가능한 상태만을 관리하는 브랜치; develop 다음 출...

https://zamoca42.github.io/blog/Etc/git-flow.html

**Comment by Zamoca42** [comment]
test
